src/collections/_documentation/apm/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Application Performance Monitoring (Beta)
+sidebar_order: 0
+---
+
+
+Something here.
```

Comment:
@MimiDumpling who is responsible for the content on this page?

Reply:
Me 😄